README.md

```diff
@@ -191,6 +191,28 @@ In JSON land, that would give you the following structure:
 Whitespace around dot-separated parts is ignored, however, best practice is to
 not use any extraneous whitespace.
 
+Defining a key multiple times is invalid.
+
+```
+# DO NOT DO THIS
+name = "Tom"
+name = "Pradyun"
```

Comment:
I added my name to the document. Hopefully, nobody minds. :')

Reply:
Perfectly fine for such a great contributor!

Reply:
Yay! Thanks!